components/alert/alert.html

```diff
@@ -13,72 +13,128 @@
         .comp-lib-hr, .comp-lib-h1, .comp-lib-h2 {
             margin: 1rem;
         }
+        .component-holder {
+            display: flex;
+            flex-direction: row;
+            align-items: flex-end;
+        }
+        .component_demo_container {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+        }
+        .code_snippets_div {
+            padding: 2rem;
+            width: fit-content;
+            margin-left: auto;
+        }
     </style>
 
 </head>
 <body>
         <!-- Alert Component Starts -->
-        <h1 class="comp-lib-h1">@ Alert Component</h1>
+        <h1 class="comp-lib-h1">Alert Component</h1>
 
-        <h2 class="comp-lib-h2">Alert Component Success</h2>
-        <!-- Alert Component Starts -- Success -->
-        <div class="alert alert--success">
-            <img class="alert__icon" src="/components/alert/success.svg" alt="success">
-            </img>
-    
-            <div class="alert__content">
-                <div class="alert__text alert__text--success">
-                    <h2 class="alert__title">Success</h2>
-                    <p class="alert__description">This is a success alert — <strong>check it out!</strong></p>
+        <h3 class="comp-lib-h2">Alert Component Success</h3>
+        <div class="component_demo_container">
+            <!-- Alert Component Starts -- Success -->
+            <div class="alert alert--success">
+                <img class="alert__icon" src="/components/alert/success.svg" alt="success">
+                </img>
+        
+                <div class="alert__content">
+                    <div class="alert__text alert__text--success">
+                        <h3 class="alert__title">Success</h3>
+                        <p class="alert__description dui-util-txt-sm">This is a success alert — <strong>check it out!</strong></p>
+                    </div>
+                </div>
+            </div>
+            <!-- Alert Component Ends -- Success -->
+            <div class="code_snippets_div">
```

Comment:
if you are following a naming convention then use the same everywhere for consistency